service/commentutil/commentutil.go

```diff
@@ -96,3 +97,59 @@ func severity(c *reviewdog.Comment) string {
 		return ""
 	}
 }
+
+// GitHubAlertComment creates a markdown comment using GitHub Alerts syntax.
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>comment on exported function GitHubAlertComemnt should be of the form "GitHubAlertComemnt ..."